android/libraries/rib-base/src/main/java/com/badoo/ribs/core/plugin/Plugin.kt

```diff
@@ -29,9 +40,11 @@ interface ViewLifecycleAware : Plugin {
 }
 
 interface SubtreeChangeAware : Plugin {
-    fun onAttachChildNode(child: Node<*>) {}
+    fun onChildCreated(child: Node<*>) {}
```

Comment:
should it be Node itself, or it can be Rib?

Reply:
Would be somewhat better with Rib, but the type is lost by the time we get here.